mon-pix/tests/acceptance/authentication/login-test.js

```diff
@@ -12,59 +12,81 @@ module('Acceptance | Login', function (hooks) {
   setupMirage(hooks);
   setupIntl(hooks);
 
-  module('when current url does not contain french tld (.fr)', function () {
-    module('when accessing the signin page with "Français" as default language', function () {
-      test('displays the signin page with "Français" as selected language', async function (assert) {
-        // when
-        const screen = await visit('/connexion');
-
-        // then
-        assert.strictEqual(currentURL(), '/connexion');
-        assert.dom(screen.getByRole('heading', { name: t('pages.sign-in.first-title'), level: 1 })).exists();
-        assert.dom(screen.getByRole('button', { name: 'Sélectionnez une langue' })).exists();
-      });
+  module('when feature toggle is false', function () {
+    server.create('feature-toggle', {
+      id: 0,
+      isNewAuthenticationDesignEnabled: false,
+    });
```

Comment:
Mettre la configuration dans un `beforeEach`

```suggestion
  module('when feature toggle is false', function (hooks) {
    hooks.beforeEach(function () {
      server.create('feature-toggle', {
        id: 0,
        isNewAuthenticationDesignEnabled: false,
      });
    });
```